### PR TITLE
adding transform_alignments

### PR DESCRIPTION
**Pull request by heuermh**
Hacked up `adam_pipeline/preprocessing.py` for this, which downloads, converts, and uploads. Early PR for feedback and because I don't remember how to build or run this stuff.

### CODE REVIEW

File: src/toil_bdg/transform_alignments/transform_alignments.py [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+from toil_lib.tools.spark_tools import call_adam, \
+    call_conductor, \
+    MasterAddress, \
+    HDFS_MASTER_PORT, \
+    SPARK_MASTER_PORT
+
+_log = logging.getLogger(__name__)
+
+
+def remove_file(master_ip, filename, spark_on_toil):
```
Comment (fnothaft) [inline src/toil_bdg/transform_alignments/transform_alignments.py:64]: You can import this from `toil_bdg.adam_pipeline.preprocessing`

Comment (fnothaft) [reply src/toil_bdg/transform_alignments/transform_alignments.py:64]: Ditto to `truncate_file`, `upload_data`, and `download_data`

**Review by fnothaft — CHANGES_REQUESTED** [review]
Few small nits. If you've got time after the BDG call tomorrow, perhaps I can get you up to speed on running it?

**Comment by heuermh** [comment]
Closing as WontFix

**Pull request closed without merging**